src/symtab/symbol_table.hpp

```diff
@@ -238,7 +238,7 @@ class SymbolTable {
  */
 class ModelSymbolTable {
     /// symbol table for mod file (always top level symbol table)
-    std::shared_ptr<SymbolTable> symtab = nullptr;
+    std::shared_ptr<SymbolTable> symtab;
```

Comment:
why this? Isn't it better to keep it explicit?

Reply:
Setting the `shared_ptr` to `nullptr` actually `reset`s the shared pointer, which is something not needed in our case, since this is its declaration